e2e/process-services-cloud/start-process-cloud.e2e.ts

```diff
@@ -92,4 +93,17 @@ describe('Start Process', () => {
         processCloudDemoPage.processListCloudComponent().checkContentIsDisplayedByName(processName);
 
     });
+
+    it('[C309875] Should display the validation message when Process definition has name property set to null', () => {
+        appListCloudComponent.checkAppIsDisplayed(simpleApp);
+        appListCloudComponent.goToApp(simpleApp);
+        processCloudDemoPage.openNewProcessForm();
```

Comment:
Can we have those 3 lines in a beforeEach ?

Reply:
we can't in this scenario, as we are doing things in afterEach to come to the main apps page and this way the test is  consistent in run results.